core/trino-main/src/main/java/io/trino/sql/planner/iterative/IterativeOptimizer.java

```diff
@@ -39,20 +39,28 @@
 import io.trino.sql.planner.plan.PlanNode;
```

Comment:
`io.trino.sql.planner.iterative.rule.ExpressionRewriteRuleSet$FilterExpressionRewrite@25b4754a` this information is not giving much as we don't know what expression rewrite is used.

Reply:
correct, it's improved in the next commit